### ISSUE TITLE
Error compiling with generated headers: to_string function

### ISSUE THREAD

**Issue by a-krebs** [ifcomp/interface-compiler#12]
Generated headers (using current develop branch at 4db3a4e21c5833262785bd950889f8626154406c) don't compile. Likely a bug in implementation of #8.

Compiler output:

```
      cannot define or redeclare 'to_string' here because namespace 'core_system' does not enclose namespace 'std'
        inline std::string std::to_string(DeviceRole enumerator)
                           ~~~~~^
```

Steps to reproduce:
1. Build compiler at 4db3a4e21c5833262785bd950889f8626154406c
2. Use this compiler to generate headers for backend project.
3. Compile backend project using generated headers.

**Comment by a-krebs** [comment]
Reverted merge of Pull Request #10 in b00c67992238cd94ef3386a38cb3bd80599d01f6 (which will need to be reverted when re-merging the branch for #8).

**Comment by vollkorn1982** [comment]
Fixed in Pull-Request #13
